src/main/java/net/minecraftforge/common/ForgeConfig.java

```diff
@@ -110,6 +110,7 @@ public static class Common {
      * Client specific configuration - only loaded clientside from forge-client.toml
      */
     public static class Client {
+        @Deprecated(forRemoval = true) // Superseded by vanilla option
```

Comment:
Please specify the version through `since`:
```suggestion
        @Deprecated(forRemoval = true, since = "1.18.2") // Superseded by vanilla option
```